script/setup.js

```diff
@@ -167,8 +167,36 @@ try {
 		}
 	};
 
+	const mainContributions = [
+		"code",
+		"content",
+		"doc",
+		"ideas",
+		"infra",
+		"maintenance",
+		"projectManagement",
+		"tool",
+	];
+
+	const addContributor = (user, contributions = []) =>
+		$`all-contributors add ${user} ${contributions.join(",")}`;
+
 	await withSpinner(
 		async () => {
+			let user;
+			try {
+				user = JSON.parse((await $`gh api user`).stdout).login;
+			} catch (err) {
+				console.warn(
+					chalk.gray(
+						"Couldn't authenticate GitHub user, falling back to the owner name you provided"
```

Comment:
[Nit] Just to make it a teeny bit more clear... what do you think?

```suggestion
						"Couldn't authenticate GitHub user, falling back to the provided owner name '${owner}'"
```